pkg/agent/secondarynetwork/cnipodcache/cache.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2021 Antrea Authors
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http:// www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cnipodcache
+
+import (
+	"sync"
+
+	"k8s.io/client-go/tools/cache"
+
+	"antrea.io/antrea/pkg/agent/util"
+	"antrea.io/antrea/pkg/util/k8s"
+)
+
+const (
+	podIndex = "pod"
+)
+
+type CNIPodInfoCache struct {
+	sync.RWMutex
+	cache cache.Indexer
```

Comment:
I feel like I am missing something obvious here, but why do we use an indexer object instead of a plain Go map, with the `"<podNamespace/podName>"` string as the key. Typically you used an indexers when you need additional ways to index your objects.

Reply:
I agree Go map is simpler here.

Reply:
sure. But, can I do it in separate PR as an improvement?.